common/djangoapps/student/tests/test_enrollment.py

```diff
@@ -218,3 +227,126 @@ def _change_enrollment(self, action, course_id=None, email_opt_in=None):
             params['email_opt_in'] = email_opt_in
 
         return self.client.post(reverse('change_enrollment'), params)
+
+    def _enrollment(self):
+        """ Generate a default course mode and enrollment. """
+        mode = CourseMode(
```

Comment:
A few issues:
1. Why are you creating a course mode here here?
2. Use `CourseMode.objects.create()`.
3. Does this belong in `setUp()`?